README.md

```diff
@@ -4,3 +4,9 @@ This repository contains packaged configuration for setting up a GKE cluster
 with [Anthos Service Mesh] features enabled.
 
 [Anthos Service Mesh]: https://cloud.google.com/anthos/service-mesh/
+
+Package Descriptions:
+
+* `asm`: Creates an ASM ready cluster and installs ASM
+* `asm-patch`: Exports an existing cluster to config, updates that cluster to
```

Comment:
Not sure what “exports an existing cluster to config” means?
Does this mean extract configuration from an existing cluster and write it to the local directory?

Reply:
Yup. I'll try to think of a better phrasing